env.example

```diff
@@ -45,15 +45,4 @@ TELEGRAM_CHAT_ID=-0000000000000
 ###############################################################################
 
 USE_IMGUR_FOR_IMAGES=true
-IMGUR_CLIENT_ID=0000000000
-
-
-###############################################################################
-#                                                                             #
-#                      Misc. required env variables                           #
-#                -- not meant for user config, don't touch --                 #
-#                                                                             #
-###############################################################################
-
-# https://github.com/yagop/node-telegram-bot-api/issues/319#issuecomment-324963294
-NTBA_FIX_319=1
+IMGUR_CLIENT_ID=7d6b00b87043f58
```

Comment:
Just curious, whose Imgur client ID is this? @Zedjones I assume?

Reply:
@jwflory this is the TeleIRC default ID.

Reply:
Right. I mean to whose account the API key is registered to. It would be good to take note.

Reply:
This API key is registered to the RITlug Imgur account. Password information is stored in RITlug's private GItLab.